### ISSUE TITLE
Infinite note/Pattern bug

### ISSUE THREAD

**Issue by Tuchan** [mat1jaczyyy/apollo-studio#238]
### Description
If you are doing a infinite note inside pattern "plugin?" and you are on cfw, when entering and exiting from setup mode the infinte notes will no longer appear. It can be fixed by going to pattern "plugin?" again and exiting it.

### To Reproduce
1. Make infinite note inside pattern.
2. Exit Pattern
3. (You need to be on cfw) Go to setup mode and then exit it.
4. Try playing any effect with infinite mode.

### How to fix
1. Go to Pattern and exit it.
2. Done

### Expected behavior
It needs to be fixed.

### Your setup
Windows 10 Apollo 1.0.0

### Additional context
if i need i can send you project in dms

**Comment by mat1jaczyyy** [comment]
Uh, device.

This happens because internally in Apollo's framebuffer, it sees that there should already be a signal with the same color there, so it does not re-output to save performance for larger effects. However when the setup is opened and closed, the Performance mode reinitializes and clears the Launchpad but the framebuffer inside Apollo stays the same.

There is no easy way to fix this, especially considering there is no way to detect when the standalone mode has changed on the Launchpad (or when setup has been closed). Unless this also happens on stock firmware, this is something that should be fixed inside of the custom firmware instead. I'll test on stock firmware later to confirm whether we should try find a way to hack around it inside Apollo or if the issue is a difference between custom and stock firmware.

**Comment by mat1jaczyyy** [comment]
Yeah, this doesn't happen on stock. This needs to be fixed in the custom firmware.
